lib/Doctrine/ORM/Decorator/EntityManagerDecorator.php

```diff
@@ -211,6 +213,20 @@ public function flush($entity = null)
         $this->wrapped->flush($entity);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function refresh($object)
+    {
+        $lockMode = null;
+
+        if (func_num_args() === 2) {
```

Comment:
```suggestion
        if (func_num_args() > 1) {
```

This might look like nit-picking, but if we want to emulate how PHP behaves if we actually had that parameter, we need to allow the caller to pass three arguments. Or fourty-two arguments.